src/components/element-card/element-card.styles.tsx

```diff
@@ -92,28 +94,39 @@ export const ActionButton = styled.button<IActionButtonStyleProps>`
 `;
 
 export const DroppableWrapper = styled.div<IDroppableWrapperProps>`
-    ${({ isOver }) =>
-        isOver
-            ? css`
-                  border: 1px dashed ${Color.Primary};
-                  border-radius: 0.25rem;
-                  background: ${Color.Accent.Light[5]};
-                  height: 100%;
-                  z-index: -1;
-              `
-            : css`
-                  border: 1px solid transparent;
-              `}
-    position: absolute;
-    flex-direction: column;
-    height: auto;
-    width: 100%;
+    position: relative;
```

Comment:
using relative positioning is causing extra space to appear below a dragged card, this is due to the `DroppableWrapper` taking up the space, it should be using `absolute` positioning

this does cause it to always take 100% width beyond its container though, to fix that, apply relative positioning to its direct parent